client/cmd/dmc/cmd/broadcast.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/GoogleCloudPlatform/cloud-datashare-toolkit/client/pkg/multicast"
```

Comment:
Should we change this to the 'datashare-toolkit' reference? The old one will still work.

Reply:
/ack